Readme.md

```diff
@@ -115,6 +115,9 @@ var nightmare = Nightmare({
 });
 ```
 
+##### ['web-preferences'].preload
+Specifies a file to add to preload.  The generated preload file is written to Electron's `userData` path.
```

Comment:
This isn't particularly clear, plus I think a user of this doesn't need to know where the generated file goes. How about:
Specifies a file to add to Electron's `preload`. (The generated preload file is written to Electron's `userData` path.)